### ISSUE TITLE
User Activity

### ISSUE THREAD

**Issue by rajatgupta24** [codeforcauseorg/edu-client#172]
## Is your feature request related to a problem? Please describe.
We need a user dashboard where the users can see the activity of themselves as well as on other's profiles (just like the GitHub activity chart), we can use [chartJS](https://www.chartjs.org/).

We can add this under the activity section of the profile page

## Additional context

Add any other context about the feature request here.

**Comment by rajatgupta24** [comment]
Hey @Abhishek-kumar09, I'd like to work on this :smiley: .

**Comment by suubh** [comment]
#135 Is it something like that ?

**Comment by rajatgupta24** [comment]
> #135 Is it something like that?

No, you can say #135 is about the GitHub profile whereas #172 is more likely to be the Github activity chart.

**Comment by Abhishek-kumar09** [comment]
Is it related to this: https://github.com/codeforcauseorg/edu-client/issues/153 or is the idea can be merged.

**Comment by rajatgupta24** [comment]
Hi @Abhishek-kumar09 can I work on this ??
